### ISSUE TITLE
question: Default authentication deny/block ?

### ISSUE THREAD

**Issue by timkeeler** [greenpau/caddy-security#78]
I'm new to caddy-security, so apologies if this is something obvious.

In the documentation example, I see how to `block` a particular email address. How would I go about implementing a default block for all users except the ones listed? I'm using Google OAuth2 and I don't want just anyone to authenticate to my page. I know I can implement `authorize`, but that still allows a user to get a token issued...

I've tried something like this:

```
transform user {
    match realm google
    match email myemail@gmail.com
    action add role authp/admin
    action add role superadmin
    ui link "Caddy Version" /version icon "las la-code-branch"
}
transform user {
    match email *
    block
}
```

but I dont' know they right way to go about it. Does transform even support wildcards? The documentation is pretty light so I dont' know what my options are...

Basically I want to `block` any authentication that isnt' coming from `myemail@gmail.com` (and I will add other users in the future).

Thanks!

**Comment by greenpau** [comment]
@timkeeler , thank you for the issue!

The transform matching is based on ACL matching https://authp.github.io/docs/authorize/acl-rbac#conditions

Have you tried “regex match email <REGEX>”?
